### DISCUSSION TITLE
Add an installer for Windows for Tuxemon

### DISCUSSION THREAD

**Discussion by Davidliang11292** [Tuxemon/Tuxemon#2294] [Show and tell]
My friend @windows11pe (no affiliation with Microsoft, I believe he just happens to be a fan of Windows) has created a Tuxemon installer for Windows and even built a webpage for it (https://windows11pe.github.io/tuxemonwindowsinstall.github.io/). This installer is compatible with Windows 7 and all subsequent versions of Windows. Although he hasn't localized this installer into English yet, he's really excited about doing so (I think it's great to make Tuxemon more accessible to Windows users). It would be fantastic if someone could try out the program and provide feedback!

**Comment by Davidliang11292** [comment]
Have a great weekend! English localization is complete! Please visit this page:
https://windows11pe.github.io/tuxemonwindowsinstall.github.io/eng/index.html

**Reply by windows11pe** [reply]
I entered the command in Linux so fast that I executed sudo rm -rf /*, so the Tuxemon-Networkinstall file I prepared was missing, and it is about to be replaced. I'm sorry

**Reply by windows11pe** [reply]
Don't ask why Linux, because I'm building Android to flash the Xiaomi MIX2S, so I can't reboot to Windows

**Reply by Davidliang11292** [reply]
While I don't know exactly what happened, oh, very good April Fool's joke, my friend.

**Reply by windows11pe** [reply]
Originally, it was sudo rm -rf /hungry.exe, which was just the program I used to test wine, because I was too lazy to put the root directory directly

**Reply by windows11pe** [reply]
I don't know if this installer will support WIndows 7 and above, because I don't have a machine with Windows 7 installed :)

**Reply by Davidliang11292** [reply]
Well, although I don't know why you would choose the root directory instead of the /home directory

**Comment by windows11pe** [comment]
This html code is quite eye-friendly
![Uploading fc13763e8f87e88955f817e54ffb5254.png…]()

**Comment by windows11pe** [comment]
I'm very worried about the domain name, because a tuxemonwindowsinstall.com domain name is very expensive, although it is not that expensive, but because I can't get it for the time being, I hope that Tuxemon can directly merge this project

**Comment by HippasusTwo** [comment]
@Davidliang11292 @windows11pe  
Alright, I guess I can be brave and download it, even though since I have the github source I don't actually need a windows installer.

Nope, it didn't work, sorry. It gave me an error that 'Tuxemon' could not be found, and that the download was aborted because the resource could not be found.
edit: Ah, it occurs to me I should probably say I have a later version of Windows, if that's relevant.

**Reply by windows11pe** [reply]
I found a problem. This should be a onedrive problem. If you want stability, you must find a server with public IPv4 or a NAS device with fixed public IPv6, because these devices will almost never be shut down for a long time and have a public network. IPv4 or fixed public network IPv6 can solve this problem
